instrumentation/opentelemetry-instrumentation-httpx/src/opentelemetry/instrumentation/httpx/__init__.py

```diff
@@ -31,7 +31,7 @@
 from opentelemetry.trace.span import Span
 from opentelemetry.trace.status import Status
 
-URL = typing.Tuple[bytes, bytes, typing.Optional[int], bytes]
+RawURL = typing.Tuple[bytes, bytes, typing.Optional[int], bytes]
```

Comment:
Does this need to be exposed outside of the module? Could it be `_RawURL` instead?

Reply:
The library defines a few internal type hints like this; they could all be private. This would be a rather large patch of unrelated changes though, so may be more appropriate as a separate PR?